DigitalInnovationHub/examples/example.json

```diff
@@ -35,13 +39,9 @@
 		"instagram": "https://www.instagram.com/lifeatsampledih"
 	},
 	"relation": [
```

Comment:
Although this is a key values file, the relation is an object, therefore subproperties have to appear.